PivotalTracker.properties

```diff
@@ -0,0 +1,7 @@
+url=https://www.pivotaltracker.com/
+
+implicitWaitChrome=30
+explicitWaitChrome=40
+
```

Comment:
do you think that we need two differents waits for the differents browsers

Reply:
Maybe we need run two scenarios in the same time with differents time waits.
Do you think that it's unncessary?

Reply:
I delete this waits you right